README.md

```diff
@@ -237,61 +239,46 @@ the output to a log file or other log management system.  See the
-	 read diagnostic information from a running agent.  This is also the host name the
-	 agent will listen on to serve internal metrics about the agent.  These metrics can
-	 can be scraped by the `internal-metrics` monitor.  This will default to `localhost`
-	 if left blank.
+```sh
+choco install signalfx-agent --params "'/access_token:YOUR_SIGNALFX_API_TOKEN'"
+```
 
- - `internalStatusPort` - This is the port that the agent will listen on so that
-	 the `signalfx-agent status` command can read diagnostic information from
-	 a running agent.  This is also the host name the agent will listen on to serve
-	 internal metrics about the agent.  These metrics can can be scraped by the
-	 `internal-metrics` monitor.  This will default to `8095`.
+The following package parameters can be set:
 
-See the section on [Privileges](#privileges) for information on the
-capabilities the agent requires.
+ * `/access_token` - The token used to send metric data to SignalFx. If not specified, the value in the `\ProgramData\SignalFxAgent\token` file (if it exists) will be used. See the **note** below for details regarding the token and starting the SignalFx Smart Agent service after installation.
+ * `/ingest_url` - Base URL of the SignalFx ingest server. If not specified, the value found in the `\ProgramData\SignalFxAgent\ingest_url` file (if it exists) will be used. Otherwise, defaults to `https://ingest.signalfx.com`.
+ * `/api_url` - Base URL of the SignalFx API server. If not specified, the value found in the `\ProgramData\SignalFxAgent\api_url` file (if it exists) will be used. Otherwise, defaults to `https://api.signalfx.com`.
+ * `/install_dir` - Installation directory. Defaults to `\Program Files\SignalFx\SignalFxAgent`.
 
-3) Run the agent by invoking the agent executable
-`SignalFxAgent\bin\signalfx-agent.exe -config <path to config.yaml>`.  By default,
-the agent logs only to stdout/err. If you want to persist logs, you must direct
-the output to a log file or other log management system.  See the
-[signalfx-agent command](./docs/signalfx-agent.1.man) doc for more information on
-supported command flags.
+The agent will look for a configuration file at
+`\ProgramData\SignalFxAgent\agent.yaml`.  If one does not already exist during
+installaion, a sample config file will be copied in place.
 
-4) You may optionally install the agent as a Windows service by invoking the
-agent executable and specifying a few command line flags.  The examples below
-show how to do install and start the agent as a Windows service.
+**Note:** If the `access_token` parameter is not specified and the
+`\ProgramData\SignalFxAgent\token` file does not exist or is empty, the SignalFx
+Smart Agent service will not automatically start after installation/upgrade.  To
+start the service, update `\ProgramData\SignalFxAgent\token` with a valid
+SignalFx token, and then either restart Windows or run the following Powershell
```

Comment:
access token.
[Not exactly true. The access token has to come from the organization to which the user wants to send data, otherwise the Agent won't work.]